features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/conf/site.json

```diff
@@ -22,6 +22,7 @@
 	    "assertionEncryptionEnabled" : "false",
         "idpInit" : "false",
         "idpInitSSOURL" : "https://localhost:9443/samlsso?spEntityID=API_PUBLISHER",
+        //"identityProviderLogoutURL" : "https://localhost:9443/samlsso"
```

Comment:
There should be a comma at the end. Otherwise just uncommenting this line will not work